dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -171,6 +172,20 @@ const podcastImageStyles = css`
 	width: 80px;
 `;
 
+const nonImmersivePodcastImageStyles = css`
+	position: absolute;
+	/**
+	* Displays 8px above the text.
+	* desired space above text (8px) - padding-top of text container (64px) = -56px
+	*/
+	bottom: -${space[14]}px;
+	left: ${space[2]}px;
+
+	${from.tablet} {
```

Comment:
DCR provides the [Hide](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/Hide.tsx#L18) component for hiding elements from certain breakpoints.